tests/server/models/test_flow_runs.py

```diff
@@ -830,6 +830,37 @@ async def test_read_flows_filters_by_deployment_id(self, flow, session):
         )
         assert {res.id for res in result} == {flow_run_1.id}
 
+    async def test_read_flow_runs_filters_by_parent_flow_run_ids(self, flow, session):
+        flow_run_1 = await models.flow_runs.create_flow_run(
+            session=session,
+            flow_run=schemas.core.FlowRun(
```

Comment:
This `schemas` module is actually the server-side schemas module (`from prefect.server import models, schemas`) so lets do two things:

- create a new test file (it's ok if it's a mirror i.e.`tests/client/api/test_flow_runs.py` and put these tests in there. Since this is a new test file we'll need to redefine a bunch of testing setup  
- update the import(s) so we can use and test from the `prefect.client.*` modules

I think the best bet would be to take a look at this test `tests/server/api/test_flow_runs.py::test_read_subflow_runs`. It's testing the server schema and I think we'll want to basically test the same way, only using the client schema instead.

Reply:
@urimandujano Thanks for the feedback :relaxed: 

I have made `tests/client/api/test_flow_runs.py` and put in the `tests/server/api/test_flow_runs.py::test_read_subflow_runs` tests, changing the filter query to use the client. When I test locally, they are passing.

In the previous pipeline, the smoke test flow failed, but this is also working just fine for me locally now. 

Thanks for your assistance, the feedback is really helpful.